libraryservice/src/main/java/com/edu/library_server/service/BookService.java

```diff
@@ -0,0 +1,31 @@
+package com.edu.library_server.service;
+
+
+import com.edu.library_server.models.Book;
+import com.edu.library_server.repository.BookRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class BookService implements IBookService {
+
+    private final BookRepository bookRepository;
+
+    @Override
+    public void saveBook(Book book) {
+        bookRepository.save(book);
+    }
+
+
+    @Override
+    public Book findBook(Integer bookID) {
```

Comment:
don't mix logic
if a method called `find()` --> just find

or u can name it by this way: `findOrCreateEmpty()`